README.md

```diff
@@ -49,14 +49,8 @@ will work:
 
     sudo pip install -r requirements.txt
 
-The Python + cmake interaction is handled in a separate project called
-[pycmake](https://github.com/Statoil/pycmake); you can either install
-that manually or use the git submodule functionality to fetch the
-correct version of `pycmake` into your *libecl* code:
+The Python + cmake interaction is handled with two cmake modules in
+`python/cmake/Modules`. These modules will be installed under
+`$prefix/share/cmake/Modules` when you install `libecl`, and they will also be
+used by the downstream modules `libres` and `ert`.
 
-    git submodule update --init pycmake
-
-
-### Installing the Python code ###
```

Comment:
How do you intend people to install the Python code? With `pip` or `setup.py` or the `CMAKE - enable python` flag?

Reply:
> How do you intend people to install the Python code? 

For now the `cmake --enable-python` is the by far best tested. At some point we should probably combine the `cmake + setup.py` based approaches. `pip install` will always only be a Python only solution - so if you are also installing `libres, ...` which utilizes the underlying C code directly, you can for the foreseeable future not use `pip`.